docs/release-notes.md

```diff
@@ -2,10 +2,40 @@
 
 Unreleased
```

Comment:
It's a draft, but as it was already reviewed, I'll just add a reminder so you don't forget to remove this line.